src/main/resources/db/migration/V1_86__juror_audit-update.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE juror_mod.juror
+    add column last_modified_by varchar(20);
+
+ALTER TABLE juror_mod.juror_audit
+    add column last_modified_by varchar(20),
+    add column last_update      timestamp;
```

Comment:
These values shouldn't be audited - we aren't auditing every property in the table but adding these this will mean we create an audit record for every update of the juror record (even if we are only updating properties which are specifically not audited e.g. responded, notes, disq_code) - this will create confusing "duplicate" records in the audit table. We only need juror personal details to be audited (name, address, dob, bank details)

Reply:
Without these values audited it will not be possible to serve up the amendment report (as we need to know who made the change :))

Reply:
Removed last_update and last_modified_by in favour for the new data within the rev_info table